doc/rst/users-guide/datapar/forall.rst

```diff
@@ -152,7 +152,7 @@ that had just overwritten the intended value in the previous iteration.
 The lesson in this example is that when using a forall-loop, the
 programmer asserts that the loop's iterations can execute safely in
 parallel and is responsible for the outcome when they cannot.  Chapel
-does not prevent nondeterministic forall loops from being written, it
+does not prevent non-deterministic forall loops from being written, it
```

Comment:
https://english.stackexchange.com/a/254385

Reply:
Read it. Agreed.